lib/associations/belongs-to-many.js

```diff
@@ -298,6 +298,46 @@ var BelongsToMany = function(source, target, options) {
 
 util.inherits(BelongsToMany, Association);
 
+BelongsToMany.prototype.get = function(instances, options) {
+  var throughModel = this.throughModel;
+
+  var throughOptions = _.assign({
+    where: _.set({}, this.foreignKey, {$in: _.pluck(instances, this.source.primaryKeyAttribute)}),
```

Comment:
Please use `instance.get` with `raw: true`, object lookups are not used internally.